src/StakeRegistry.sol

```diff
@@ -245,6 +248,11 @@ contract StakeRegistry is StakeRegistryStorage {
             _strategyParams[indicesToRemove[i]] = _strategyParams[_strategyParams.length - 1];
             _strategyParams.pop();
         }
+
+        // If no strategies exist in quorum, turn bit off 
+        if (_strategyParams.length == 0) {
+            quorumBitmap = uint192(quorumBitmap.minus(uint192(1) << quorumNumber));
```

Comment:
Would prefer a helper for this. Could use something similar to the `setBit` method I linked, but with a bool on/off switch?

Reply:
Or just `unsetBit` :shrug: